nixos/modules/services/mail/roundcube.nix

```diff
@@ -95,6 +95,14 @@ in
       '';
     };
 
+    maxAttachmentSize = mkOption {
+      type = types.int;
+      default = 18;
+      description = ''
+        The maximum attachment size in MB. Note: The value is not 100% accurate since rounducbe uses a percentage of php upload configuration.
```

Comment:
You should probably document the `* 1.3` calculation here to make it clear which value to set.

Reply:
I changed the description. Feel free to propose something else if you've some better ideas.